modules/fs2/src/main/scala/precog/contrib/fs2/io.scala

```diff
@@ -17,13 +17,17 @@
 package precog.contrib.fs2
 
 import cats.effect.Sync
+import cats.implicits._
 import fs2.Stream
 
-object IO {
+object io {
+
   def resourceAsBytes[F[_]: Sync](
       resourceName: String,
       cl: ClassLoader = getClass.getClassLoader,
       chunkSize: Int = 32 * 1024): Stream[F, Byte] =
-    fs2.io.readInputStream(Sync[F].delay(cl.getResourceAsStream(resourceName)), chunkSize)
-
+    Stream.eval(Sync[F].delay(Option(cl.getResourceAsStream(resourceName)))).flatMap {
```

Comment:
⛳I think this could work 
```suggestion
    fs2.io.readInputStream(Sync[F].defer(Option(cl.getResourceAsStream(resourceName)).liftTo[F](new Exception(s"Resource not found: $resourceName"))))
```

Reply:
Nice, thank you!